packages/dx-react-chart/demos/featured/olympic-games.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Olympic Games
+[BarSeries](https://devexpress.github.io/devextreme-reactive/react/chart/docs/reference/bar-series/)
+This demo shows the Chart component that displays a medal amount of the Olympic games with a BarSeries.
```

Comment:
For pass a reference we often use next syntax
`This demo shows the Chart component that displays a medal amount of the Olympic games with a
[BarSeries](https://devexpress.github.io/devextreme-reactive/react/chart/docs/reference/bar-series/)`